sdk/src/main/java/io/dapr/actors/runtime/ActorStatefulService.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import io.dapr.actors.ActorId;
+
+/**
+ * Implementation of the Actor Service that contains a state provider.
+ */
+class ActorStatefulService implements ActorService {

```

Comment:
I don't know if we want to name it this.  "stateful" is a bit of a strong word, and the existence of the state provider doesn't mean they actually have to store state.  What about ActorServiceImpl?

Reply:
Agree in name change.

Reply:
Will do. I had it names ActorServiceImpl but I am not sure about the Impl convention yet.